src/main/kotlin/no/nav/familie/ef/mottak/task/HentSaksnummerFraJoarkTask.kt

```diff
@@ -18,15 +16,7 @@ class HentSaksnummerFraJoarkTask(private val taskRepository: TaskRepository,
                                  private val hentJournalpostService: HentJournalpostService) : AsyncTaskStep {
 
     override fun doTask(task: Task) {
-        try {
-            hentJournalpostService.hentSaksnummer(task.payload)
-        } catch (notFound: HttpClientErrorException.NotFound) {
-            LOG.info("Hent saksnummer returnerte 404 response body={}",
-                     notFound.responseBodyAsString)
-            val copy = task.copy(triggerTid = LocalDateTime.now().plusMinutes(15))
-            taskRepository.save(copy)
-            throw notFound
```

Comment:
Usikker på om jeg skal slette dette. Hva tenker dere?

Reply:
Det må du nok ha med. Kan ta opp til et døgn før BRUT001 blir kjørt. Uten denne så vil de havne i feilet etter 3 forsøk og ikke komme videre før feilede tasker rekjøres. Som gjøres 1 gang i døgnet

Reply:
Må også ha med maxAntallFeil

Reply:
Har tatt tilbake try/catch. maxAntallFeil er også satt for denne typen task. Notert at metadata må legges inn i Properties, men gjør ikke dette nå.

Reply:
La til oppgaver i Trello for å sjekke logging og oppdatering av metadata i Properties.